packages/pangea-sdk/tests/integration/test_authn.py

```diff
@@ -160,7 +160,7 @@ def test_authn_a5_user_update(self):
         self.assertEqual(False, response.result.disabled)
 
     def test_authn_b1_user_invite(self):
-        # This could (should) fail if test_authn_user_create_with_password failed
+        # This could (should) fail if test_authn_user_createf_with_password failed
```

Comment:
I don't see anything with either of these names.

Reply:
fixed